tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
 envlist =
-    py27-codestyle,
-    py27
+    py36-codestyle,
+    py36
 
 [testenv]
 install_command = pip install --no-deps {opts} {packages}
 deps = -r{toxinidir}/requirements.txt
 commands = pytest {posargs}
 
-[testenv:py27-codestyle]
+[testenv:py36-codestyle]
```

Comment:
you need to copy/paste this so we have both py27-codestyle and py36-codestyle configured